guess_the_words/function.py

```diff
@@ -0,0 +1,161 @@
+import json
+import string
+import random
+import datetime
+
+
+def create_list():
+    with open('words.txt', 'r') as file:
+        list_words = file.readlines()
+        for posit, word in enumerate(list_words):
+            list_words[posit] = word.strip()
+    return list_words
+
+
+def val_count(player_letters: list) -> list:
+    if len(player_letters) > 5:
+        print('You enter word more then 5 letters')
+        return True
+    elif len(player_letters) < 5:
+        print('You enter word less then 5 letters')
+        return True
```

Comment:
```suggestion
        return True

    return False
```

Reply:
I have removed two functions - I discovered a bug in which, if you enter less than five letters, a message is written that you entered less than 5 letters and the program shows whether they were guessed.
Replaced with the code.